src/routes/about/__tests__/get-about.e2e-spec.ts

```diff
@@ -4,19 +4,28 @@ import * as request from 'supertest';
 import { AppModule } from '@/app.module';
 import { expect } from '@jest/globals';
 import '@/__tests__/matchers/to-be-string-or-null';
+import { CacheKeyPrefix } from '@/datasources/cache/constants';
 
 describe('Get about e2e test', () => {
   let app: INestApplication;
 
   beforeAll(async () => {
+    const cacheKeyPrefix = crypto.randomUUID();
     const moduleRef = await Test.createTestingModule({
       imports: [AppModule.register()],
-    }).compile();
+    })
+      .overrideProvider(CacheKeyPrefix)
+      .useValue(cacheKeyPrefix)
+      .compile();
 
     app = moduleRef.createNestApplication();
     await app.init();
   });
 
+  afterAll(async () => {
+    await app.close();
+  });
```

Comment:
I've moved the `afterAll` to the section before each test. In my opinion this makes it easier to see what is done before and after each test (as we can have quite a few tests before we get to the `afterEach`, `afterAll`)